tornettools/plot.py

```diff
@@ -192,9 +208,13 @@ def __plot_circuit_build_time(args, torperf_dbs, tornet_dbs):
 
     __plot_cdf_figure(args, dbs_to_plot, 'circuit_build_time',
```

Comment:
I think you want `f'circuit_build_time.{circuittype}'` instead of just `'circuit_build_time'`.

Reply:
done